open_schools_platform/testing_examples/migrations/0001_initial.py

```diff
@@ -28,7 +28,7 @@ class Migration(migrations.Migration):
                 ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('email', models.EmailField(max_length=255)),
                 ('identifier', models.UUIDField(default=uuid.uuid4)),
-                ('school', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='students', to='testing_examples.school')),
+                ('school', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='student', to='testing_examples.school')),
```

Comment:
it is not ok
perhaps when you refactored student this migration was also changed but it shouldn't

Reply:
return as it was